src/txkube/_authentication.py

```diff
@@ -244,7 +245,10 @@ def https_policy_from_config(config):
         signed by the certificate authority certificate associated with the
         active context's cluster.
     """
-    base_url = URL.fromText(config.cluster["server"].decode("ascii"))
+    server = config.cluster["server"]
+    if isinstance(server, bytes):
+        server = server.decode("ascii")
+    base_url = URL.fromText(server)
```

Comment:
I think this is at least the 2nd instance of this pattern.  Time for a helper?

Reply:
I came up with a helper here: https://github.com/LeastAuthority/txkube/pull/172

Reply:
Now that https://github.com/LeastAuthority/txkube/pull/173 is merged, there is only 1 instance of this pattern.  Do you still want a helper?